### ISSUE TITLE
[Suggestion] Option to configure (or disable) Brimstone explosion radius

### ISSUE THREAD

**Issue by MarioSMB** [Sweenus/SimplySwords#22]
I would like to be able to use the Brimstone Claymore without causing creeper explosions constantly! An option to disable brimstone griefing or at least adjust the radius of it would be much appreciated, thanks!  

P.S. In the 1.18.2 effects_config.json5 file, the brimstone settings appear to be a mismatch with some bramble effect:  
```
// -- Unique Effect: Brimstone -- 
// ------------------------------ 
  "brimstone_chance": 15.0,
  "bramble_chance": 45.0,
  "bramble_radius": 10.0
```

**Comment by Sweenus** [comment]
Thanks for the suggestion - I'll take a look at the Brimstone behaviour.
That config issue is fixed in 1.19. I'm undecided whether the 1.18 build will be recieving an update at this stage, but if it does that config issue will be fixed :)

**Comment by Sweenus** [comment]
I've reworked the Brimstone Claymore ability a bit, and it will no longer cause terrain damage.
This will be live in the next update. Thank you again for your feedback!
